jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e-gce.yaml

```diff
@@ -25,6 +25,7 @@
         - activate-gce-service-account
         - shell: |
             {provider-env}
+            {project-env}
```

Comment:
I think you need to define a default `project-env` for all of the projects that don't use override it. (it can probably be an empty string defined in this job-template.)